docs/users/games/StardewValley.md

```diff
@@ -1,2 +1,44 @@
-!!! tip "Coming Soon"
-    Stardew Valley is not supported yet, but will be one of the next games we add to the app. Stay tuned!
\ No newline at end of file
+Use the Nexus Mods app to manage your Stardew Valley mods!
+
+## Features
+On top of the basic mod management features, players also benefit from these dedicated features:
+
+### SMAPI Installer
+Manage your installatioon of Stardew Modding API (SMAPI) by simply downloading and installing it as you would with any other mod. Once installed, your SMAPI version and game version are compared to ensure that they are compatible. 
+
+### Loadout Health Check
+Get information on potential issues in your loadout(s). 
+
+![An example Health Check message prompting the installation of a missing mod.](../images/0.6.0/HealthCheckExample.webp)
+
+Diagnostics are shown in the following situations: 
+
+- SMAPI is recommended when no mods are installed yet.
+- The installed SMAPI version is too old to work with the game version.
+- The game version is too old to work with the installed SMAPI version.
+- An installed mod's dependencies are installed, but they are outdated.
+- An installed mod has missing dependencies.
+- An installed mod's dependencies are installed, but they are disabled.
+- A mod that requires SMAPI is installed but SMAPI is missing.
+- A mod that requires SMAPI is installed but SMAPI is disabled.
+- A mod is marked as obsolete or broken (according to SMAPI).
+- A mod is overwriting the game files (XNB mod).
```

Comment:
We plan on generating documentation for all diagnostics based on the code definitions in the future. That would keep the docs up-to-date with the code.

For now, I think it's enough to say that we have diagnostics, and maybe a list of some examples instead of a trying to put a full list in the docs. This list is already outdated. See [the definitions](https://github.com/Nexus-Mods/NexusMods.App/blob/9712e30cc61f61ba69c881a9dd5b8586dc83a456/src/Games/NexusMods.Games.StardewValley/Diagnostics.cs) for a complete list.

Reply:
Without this list the feature set looks pretty sparse so I'm fine updating this by hand for now.